src/load/argdown.ts

```diff
@@ -0,0 +1,114 @@
+import * as date from "../date.js";
+import * as model from "../model/index.js";
+import * as argdownSchema from "../schemas/argdown.js";
+import {
+  ArgdownApplication,
+  IArgdownRequest,
+  ParserPlugin,
+  ModelPlugin,
+  PreselectionPlugin,
+  MapPlugin,
+  IMapNode,
+  IMapEdge,
+} from "@argdown/core";
+
+export function argdown(obj: string) {
+  const app = new ArgdownApplication();
+
+  const parserPlugin = new ParserPlugin();
+  app.addPlugin(parserPlugin, "parse-input");
+
+  const modelPlugin = new ModelPlugin();
+  app.addPlugin(modelPlugin, "build-model");
+
+  const preselectionPlugin = new PreselectionPlugin();
+  app.addPlugin(preselectionPlugin, "preselection-plugin");
+
+  const mapPlugin = new MapPlugin();
+  app.addPlugin(mapPlugin, "map-plugin");
+
+  const request: IArgdownRequest = {
```

Comment:
Are all of these plugins required? I only had a brief look at the code of `argdown/core`, but to me it seems that the [JSONExportPlugin](https://argdown.org/argdown-core/classes/plugins_JSONExportPlugin.JSONExportPlugin.html) has less required plugins. Could you investigate the impact on the bundle size by running `npm build`?

Reply:
To use the MapPlugin the other plugins are required. For the JSONExportPlugin the ModelPlugin and the ParserPlugin would be needed. The reason why I used the MapPlugin is that it provides easier access to the graph edges and nodes.

Reply:
Ok, makes sense. Would still be interesting to check the bundle size: If the package becomes rather large, we may make the argdown dependency optional.